contractcourt/channel_arbitrator.go

```diff
@@ -1122,7 +1118,7 @@ func (c *ChannelArbitrator) sweepAnchors(anchors []*lnwallet.AnchorResolution,
 			&anchorInput,
 			sweep.Params{
 				Fee: sweep.FeePreference{
-					ConfTarget: anchorSweepConfTarget,
+					ConfTarget: anchor.CommitDeadline,
```

Comment:
After more thought, the `ConfTarget` should stay as it is and add a new field `Deadline` to the `sweep.Params`. `ConfTarget` essentially expresses the "expected value", while `Deadline` is our end time.

Reply:
Not sure I follow, If `Deadline` is set, how would `ConfTarget` be used in practice?

Reply:
Alos note that the deadline is absolute block height here, while the conf_target is relative

Reply:
> Not sure I follow, If `Deadline` is set, how would `ConfTarget` be used in practice?

After a second thought, yeah I think they would mean the same thing.

> Alos note that the deadline is absolute block height here, while the conf_target is relative

Yeah I made a mistake here, will change.